docs/ref/templates/api.txt

```diff
@@ -99,7 +99,7 @@ overridden by what's passed by
       * ``'django.template.loaders.app_directories.Loader'`` if and only if
         ``app_dirs`` is ``True``.
 
-      If ``debug`` is ``False``, these loaders are wrapped in
+      These loaders are then wrapped in
```

Comment:
This is far away from the `versionchanged` annotation, so I would add another one here.